socrates/lib/site/views/format.jade

```diff
@@ -0,0 +1,170 @@
+        It can really be anything the group want it to be.
+      p.
+        And this is the beauty of open space - it really will be anything the
+        group wants it to be. You are feeling a bit frustrated with something
+        at work? How about a therapy session outside in the sun with a few
+        like-minded people over a glass of wine or a cup of tea? You have a
+        particular question about something you're stack with? Just ask for
+        help. You... get the picture now. :)
+      p.
+        And this brings us to...
+
+      h3 People
+      p.
+        SoCraTes is all about people. There are going to be 200 motivated,
+        intelligent and friendly participants, just like yourself, to make the
+        best of the time together.
+      p.
+        You'll share breakfast, lunch and dinner together. Some of the best
+        conversations happen over a meal. You'll have evenings together in a
+        bar, play games together, go for a run or early walk together. There's
+        definitely going to be a game or 100 of werewolf, fun rounds of beach
+        volleyball and much more.
+      p.
+        We all stay at the same location for the sole reason of having enough
+        time with each other - enough time to get to know each other, learn
+        from each other, create business contacts, recuperate from the real
+        world and have generally great time. If you've never been to a
+        conference like this, it will be an eyeopener. No more divisions
+        between speakers and audience, going home alone or set schedule.
+        Doesn't matter if you're a world-class expert, world-weary salary-man,
+        a student, or have just started to program a couple of months ago.
+        Everybody has something to give, to share and to learn from others. And
+        all of that happens in a friendly, supportive and safe environment that
+        embraces and celebrates diversity.
+      p.
+        This can sound a bit unusual, but it works beautifully in practice.
+        Why? Because so many amazing people are at the same place, focusing on
+        each other. By the end of the conference you'll know by name a few
+        dozens of new faces. You'll have made friends, business connections if
+        you want them, maybe you'll find a mentor, or a kindred spirit for that
+        interesting project you want to work on. Anything can happen. And is
+        bound to happen.
+      p
+        | And
+        b  that
+        |  is what SoCraTes is really all about.
+
+      h3 Schedule
+      p.
+        So now that you know what SoCraTes actually is, here are a few more
+        details about the schedule.
+      dl
+
+        dt Thursday
+        dd.
+          The conference starts at 5pm on Thursday, August 25th 2016.
+          This is the time to get to know new faces or meet old friends, have
+          some drinks at the bar, explore the surrounding area etc.
```

Comment:
surrounding area -> location
(we don't want them to start a hike straight away :-) )